src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities/Utilities.cs

```diff
@@ -34,5 +34,10 @@ public static KeyValueLogScope CreateArtifactScope(string artifactType, IEndpoin
             scope.AddArtifactEndpointInfo(endpointInfo);
             return scope;
         }
+
+        public static ProcessKey? GetProcessKey(int? pid, Guid? uid, string name)
+        {
+            return (pid == null && uid == null && name == null) ? null : new ProcessKey(pid, uid, name);
```

Comment:
I think you should use `.HasValue` and `string.IsNullOrEmpty(name)` to be consistent with the rest of the code.